internal/graph/graph.go

```diff
@@ -167,21 +167,31 @@ func (g *ConnectedObjectGraph) findIngressesWithRewrite(
 
 		return res, nil
 	case *openfgapb.Userset_ComputedUserset: // e.g. target = define viewer as writer
+
+		var ingresses []*RelationshipIngress
+
 		// if source=document#writer
 		if target.GetType() == source.GetType() && t.ComputedUserset.GetRelation() == source.GetRelation() {
```

Comment:
this line and line 214 are very similar, any chance of extracting it into a function that has a meaningful name?

Reply:
Renamed the variables to `sourceRelMatchesRewritten` because this condition indicates that the source relationships matches the rewritten relationship (on the computed_userset).